content/docs/setup/customize/aws-elb/alb.md

```diff
@@ -0,0 +1,185 @@
+---
+title: AWS ALB
+weight: 10
+---
+
+In this guide you explore how to expose the {{< reuse "docs/snippets/product-name.md" >}} proxy with an AWS application load balancer (ALB). 
+
+{{< callout type="warning" >}}
+The AWS Load Balancer Controller only supports creation of an ALB through an Ingress Controller and not through the {{< reuse "docs/snippets/k8s-gateway-api-name.md" >}}. Because of this, you must create the ALB separately through an Ingress resource that connects it to the service that exposes your gateway proxy.
+{{< /callout >}}
+
+## Before you begin
+
+1. Create or use an existing AWS account. 
+2. Follow the [Get started guide](/docs/quickstart/) to install {{< reuse "docs/snippets/product-name.md" >}} and deploy the httpbin sample app. You do not need to set up a Gateway as you create a custom Gateway as part of this guide. 
+   
+## Step 1: Deploy gateway proxy resources
```

Comment:
do we still have the steps to deploy the AWS LB controller?

Reply:
yes, in the conref on line 103